config.m4

```diff
@@ -491,6 +498,9 @@ if test "$MONGODB" != "no"; then
     AC_SUBST(MONGOC_VERSION)
     AC_OUTPUT($srcdir/src/libmongoc/src/mongoc/mongoc-config.h)
     AC_OUTPUT($srcdir/src/libmongoc/src/mongoc/mongoc-version.h)
+    if test "x$with_zlib" = "xbundled"; then
+      AC_OUTPUT($srcdir/src/libmongoc/src/zlib-1.2.11/zconf.h)
```

Comment:
`zconf.h.in` actually doesn't have an substitutions, but we need this line to ensure it's copied to `zconf.h`, which `zlib.h` includes.

Reply:
Make sure to add it to .gitignore though.

Reply:
https://github.com/mongodb/mongo-c-driver/blob/1.9.2/.gitignore#L119